constitution.tex

```diff
@@ -254,7 +254,7 @@ \subsection{Modifying the Constitution}
 
 \asubsection{Active Membership Expectations}
 Active Members are expected to be active participants in CSH as defined in \ref{Expectations of Active Members}.
-Active Voting Members are expected to meet the requirements defined in \ref{Expectations of Voting Members}.
+Active Voting Members are eligible to become Voting Members according to the requirements defined in \ref{Qualifications of Voting Members}.
```

Comment:
How about:

Active Members are eligible to become Active Voting Members if they meet the requirements defined in \ref{Qualifications of Voting Members}.

Reply:
(This change ensures we use only existing definitions.)